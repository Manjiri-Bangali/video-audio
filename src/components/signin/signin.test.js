import { render } from "@testing-library/react";
import Signin from './signin';

describe("SigninComponent Tests", () => {
    it("Signin renders successfully", () => {
        const signinComponent = render(<Signin/>);
        expect(signinComponent).toMatchSnapshot();
    });
    
});