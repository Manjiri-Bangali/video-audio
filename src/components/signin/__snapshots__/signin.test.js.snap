// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SigninComponent Tests Signin renders successfully 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="signin-component"
      >
        <div
          class="shapes yellow-circle"
        />
        <div
          class="shapes green-circle"
        />
        <div
          class="shapes red-circle"
        />
        <h3>
          Sign In
        </h3>
        <div
          class="content"
        >
          <form>
            <div>
              <label
                class="input-label"
                for="email"
              >
                Email address
              </label>
              <input
                id="email"
                placeholder="Enter your Email address"
                type="text"
              />
            </div>
            <div
              class="password-panel"
            >
              <label
                class="input-label"
                for="password"
              >
                Password
              </label>
              <a
                class="forgot-label"
                href="#"
              >
                Forgot?
              </a>
              <input
                id="password"
                placeholder="Enter your password"
                type="password"
              />
            </div>
          </form>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="signin-component"
    >
      <div
        class="shapes yellow-circle"
      />
      <div
        class="shapes green-circle"
      />
      <div
        class="shapes red-circle"
      />
      <h3>
        Sign In
      </h3>
      <div
        class="content"
      >
        <form>
          <div>
            <label
              class="input-label"
              for="email"
            >
              Email address
            </label>
            <input
              id="email"
              placeholder="Enter your Email address"
              type="text"
            />
          </div>
          <div
            class="password-panel"
          >
            <label
              class="input-label"
              for="password"
            >
              Password
            </label>
            <a
              class="forgot-label"
              href="#"
            >
              Forgot?
            </a>
            <input
              id="password"
              placeholder="Enter your password"
              type="password"
            />
          </div>
        </form>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
